airflow/dag_processing/bundles/manager.py

```diff
@@ -64,6 +64,23 @@ def parse_config(self) -> None:
                 "Bundle config is not a list. Check config value"
                 " for section `dag_bundles` and key `backends`."
             )
+
+        # example dags!
+        if conf.getboolean("core", "LOAD_EXAMPLES"):
+            from airflow import example_dags
+
+            example_dag_folder = next(iter(example_dags.__path__))
+            backends.append(
+                {
+                    "name": "example_dags",
```

Comment:
Should we prevent people from having a bundle named example_dags in the configuration?

Reply:
Also, what should we do about this: https://github.com/apache/airflow/blob/a149dd72b31cf428e3a87b2ab3e3197cadd9a014/airflow/utils/file.py#L270-L274

Reply:
> Should we prevent people from having a bundle named example_dags in the configuration?

Yes! Good call, I'll do that in a followup.

> what should we do about this:

That's removed in #45532 - I unfortunately can't remove that in this PR as it'll break the example DAGs in the meantime.

Reply:
See #45566 for the name check.